api_review/ColorPickerDisplayMode/ColorPickerDisplayModeSpec.md

```diff
@@ -0,0 +1,118 @@
+> See comments in Markdown for how to use this spec template
+
+<!-- The purpose of this spec is to describe a new feature and
+its APIs that make up a new feature in WinUI. -->
+
+<!-- There are two audiences for the spec. The first are people
+that want to evaluate and give feedback on the API, as part of
+the submission process.  When it's complete
+it will be incorporated into the public documentation at
+docs.microsoft.com (http://docs.microsoft.com/uwp/toolkits/winui/).
+Hopefully we'll be able to copy it mostly verbatim.
+So the second audience is everyone that reads there to learn how
+and why to use this API. -->
+
+# Background
+<!-- Use this section to provide background context for the new API(s) 
+in this spec. -->
+
+<!-- This section and the appendix are the only sections that likely
+do not get copied to docs.microsoft.com; they're just an aid to reading this spec. -->
+
+<!-- If you're modifying an existing API, included a link here to the
+existing page(s) -->
+
+<!-- For example, this section is a place to explain why you're adding this API rather than
+modifying an existing API. -->
+
+<!-- For example, this is a place to provide a brief explanation of some dependent
+area, just explanation enough to understand this new API, rather than telling
+the reader "go read 100 pages of background information posted at ...". -->
+This API is an update to the existing ColorPicker control. 
+The aim of this new API is, to make the ColorPicker more versatile to use and provide different layouting ways to maximize the space available.
+
+# Description
+<!-- Use this section to provide a brief description of the feature.
+For an example, see the introduction to the PasswordBox control 
+(http://docs.microsoft.com/windows/uwp/design/controls-and-patterns/password-box). -->
+
+With the DisplayMode API for the ColorPicker, the ColorPicker is able to either layout itsself horizontally or vertically. 
+In addition to this the ColorPicker can also choose its layout based on the space available.
```

Comment:
This auto behavior doesn't feel strictly necessary to me, and omitting it would, I suspect, greatly simplify the implementation. 

I read the original proposal as giving app developers a choice of whether the control should lay itself out horizontally or vertically. So it feels ok to enable that choice without trying to do anything clever automatically.

Reply:
Seems like a reasonable idea. If an "Auto" mode is really necessary, developers could achieve it using VisualStateTriggers or we can add it later. I removed the "Auto" mode from the spec proposal.